src/styles/card.css

```diff
@@ -0,0 +1,102 @@
+.card {
+  font-family: inherit;
+
+  width: 200px;
+  height: 406px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: block;
+  height: 134px;
+  width: 160px;
+
+  margin: 0 auto;
+}
+
```

Comment:
margin & padding it's also a block model, so keep it together

Reply:
**Fixed**